shared/i18n/i18n.js

```diff
@@ -0,0 +1,31 @@
+import i18n from "i18next";
+import * as language from "./lang";
+
+i18n.init({
+    // lng: "en", // 'en' | 'es'
+    // Using simple hardcoded resources for simple example
+    resources: {
+        en: language.en,
+        fr: language.fr,
+        pl: language.pl
+    },
```

Comment:
Please loop through the langauge and set the resources. That way, we don't have to touch is file again.
```
const resources = {};
Object.keys(language).forEach(lang => {
       resources[lang] = language[lang];
}
```